src/app/app.component.scss

```diff
@@ -1,3 +1,11 @@
 .skyux-app-loading {
   visibility: hidden;
 }
+
+/deep/
+.sky-modal-body-full-page {
+  // Hide the bb-Help-invoker when a full-page modal is present and the widget is closed.
+  #bb-help-container.closed > #bb-help-invoker {
+    display: none;
```

Comment:
@Blackbaud-BobbyEarl What are your thoughts on adding an integration test for this behavior?
https://github.com/blackbaud/skyux-builder/tree/master/e2e

My thoughts: if we add a test, we could catch any selector changes from either the widget or UX.

Reply:
I agree @Blackbaud-SteveBrush.  I talked to @Blackbaud-BrandonJones.  I'm comfortable getting this into builder right now due time constraints.  He's added a task to his backlog to work on a test for this scenario, as well as a few other help related scenarios.